packages/contract-addresses/addresses.json

```diff
@@ -20,7 +20,7 @@
         "devUtils": "0x74134cf88b21383713e096a5ecf59e297dc7f547",
         "erc20BridgeProxy": "0x8ed95d1746bf1e4dab58d8ed4724f1ef95b20db0",
         "uniswapBridge": "0x36691c4f426eb8f42f150ebde43069a31cb080ad",
-        "erc20BridgeSampler": "0xc0154b14cc60a6661171fdc817f759429a57e184",
+        "erc20BridgeSampler": "0xd7223e59a11bbeb663c77625f06963ff42fdf97d",
```

Comment:
Can we get this source verified or redeploy if the artifact is lost prior to merge?

Reply:
I'll redeploy, verify, and update before merging